code/modules/clothing/clothing.dm

```diff
@@ -9,6 +9,7 @@
 	var/flash_protect = FLASH_PROTECTION_NONE
 	var/tint = 0 //Sets the item's level of visual impairment tint, normally set to the same as flash_protect
 	var/up = FALSE //but separated to allow items to protect but not impair vision, like space helmets
+	var/pepper_protect = FALSE //Toggles pepper protection when an item is ajusted up/down
```

Comment:
You also shouldn't *need* a whole var for this, when a visor toggles it uses `^=`, which literally toggles the bit on or off